WifiART/src/structs/BssidState.java

```diff
@@ -0,0 +1,26 @@
+package structs;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class BssidState {
```

Comment:
Javadocs needed.

Reply:
+1. Not sure what BssidState is supposed to represent.